src/parsers/poap.js

```diff
@@ -8,7 +7,7 @@ import { imageToPng } from '@rainbow-me/handlers/imgix';
  */
 
 export const parsePoaps = data => {
-  const poaps = get(data, 'data', null);
+  const poaps = data?.data ?? null;
```

Comment:
is it ok for this function to throw? i mean on the very next line it assumes that poaps is an array

Reply:
It's handled in upper try-catch. For now, it's not broken anything, but it should be changed definitely